promitor-agent-scraper/README.md

```diff
@@ -60,8 +60,10 @@ their default values.
 | `image.tag`  | Tag of image to use | None, chart app version is used by default            |
 | `image.pullPolicy`  | Policy to pull image | `Always`            |
 | `image.pullSecrets`  | ImagePullSecrets for the pod | `[]`            |
-| `azureAuthentication.appId`  | Id of the Azure AD entity to authenticate with |             |
-| `azureAuthentication.appKey`  | Secret of the Azure AD entity to authenticate with |             |
```

Comment:
We'll have to be backward compatible here where the new one is favored over the old one; but we shouldn't break the current experience.

We can thus keep this but mark it as deprecated with reference to new appraoch